tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -936,7 +936,7 @@ def test_sessions_without_users(self):
         )
 
         assert response.status_code == 200, response.content
-        assert result_sorted(response.data)["groups"] == [
+        assert response.data["groups"] == [
```

Comment:
This test passed by accident because I used `result_sorted`. What we actually want to test here is the order of groups, which is erased by `result_sorted`.